contract/test/snapshots/test-start1.js.md

```diff
@@ -0,0 +1,62 @@
+# Snapshot report for `test/test-start1.js`
+
+The actual snapshot is saved in `test-start1.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## use contractStarter to start postalSvc
+
+> Terms of contractStarter and the contracts it starts are published under boardAux
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.boardAux.board0371',
+        {
+          terms: {
+            agoricNames: Object @Alleged: NameHubKit nameHub {},
+            board: Object @Alleged: Board board {},
+            brands: {
+              Fee: Object @Alleged: ZDEFAULT brand {},
+              Invitation: Object @Alleged: Zoe Invitation brand {},
```

Comment:
a [marshaller that shows slots](https://github.com/Agoric/ui-kit/pull/72) would help here